solver/llbsolver/mounts/mount_test.go

```diff
@@ -288,8 +289,8 @@ func TestCacheMountLockedRefs(t *testing.T) {
 	gotRef4 := make(chan struct{})
 	go func() {
 		ref4, err := g2.getRefCacheDir(ctx, nil, "foo", pb.CacheSharingOpt_LOCKED)
-		require.NoError(t, err)
-		require.Equal(t, ref.ID(), ref4.ID())
+		assert.NoError(t, err)
```

Comment:
Why is this sometimes changed from `assert.NoError` to `require.NoError` and other times it's the opposite? Is this because this is in a goroutine?

Reply:
Yes, the rule is that require is not allowed in goroutines, same as https://pkg.go.dev/testing#B.FailNow